lib/Core/loadSdmxDataJson.js

```diff
@@ -0,0 +1,13 @@
+const Resource = require("terriajs-cesium/Source/Core/Resource");
+
+function loadSdmxDataJson(urlOrResource) {
```

Comment:
Because this is so specific to SDMX and is only used it one place, it'd be fine to just use this code inline where it's needed rather than creating a separate function. Really even the `loadJson` function exists primarily because a similar function used to exist in Cesium and it was deprecated, and it was easier to add our own version of it than to change all our call sites. No need to go back and change this unless you feel like it, just an FYI.